docs/v1/get-ticket-data/get-most-ticketed-games.md

```diff
@@ -81,8 +81,9 @@ if (response is NetworkResponse.Success) {
 			"GameIcon": "\/Images\/081550.png",
 			"Console": "PlayStation",
 			"OpenTickets": 11
-		},]
-    "URL": "https://retroachievements.org/ticketmanager.php?f=1"
+		},
+  ],
```

Comment:
Looks like something weird may be going on with the indentation here.

Reply:
Sorry about that, I had to figure out how to run the formatter